orx-rabbit-control/build.gradle

```diff
@@ -20,6 +20,7 @@ dependencies {
     implementation "io.github.rabbitcontrol:rcp:0.3.1"
     implementation "com.google.zxing:core:3.4.0"
     implementation "com.google.zxing:javase:3.4.0"
+    compile "io.ktor:ktor-server-netty:1.3.2"
```

Comment:
`implementation` is the advised way to do this. `compile` is deprecated

Reply:
Ah ok